website/docs/docs/build/environment-variables.md

```diff
@@ -92,9 +92,27 @@ While all environment variables are encrypted at rest in dbt Cloud, dbt Cloud ha
 
 ### Special environment variables
 
-dbt Cloud has a number of pre-defined variables built in. The following environment variables are set automatically for deployment runs, and their values cannot be changed.
+dbt Cloud has a number of pre-defined variables built in. Variables are set automatically and cannot be changed.
```

Comment:
@mirnawong1 I moved this up because it feels like it applies to all special env variable, but should it be versioned?

If it does belong only under "dbt Cloud IDE details" then I think we need a sentence up here to describe "special env variables"